docs/bdd/basic-usage.rst

```diff
@@ -28,12 +32,18 @@ Scenario::
             Then I should be notified that it has been successfully created
             And the country "France" should appear in the store
 
-Pay attention to the form of these sentences. From the developer point of view they are hiding the details of the feature's implementation. Instead of describing "When I click on the select box And I choose France from the dropdown Then I should see the France country in the table" - we are using sentences that are less connected with the implementation, but more focused on the effects of our actions.
-A side effect of such approach is that it results in steps being really generic, therefore if we want to add another way of testing this feature for instance in the domain or api context, it will be extremely easy to apply. We just need to add a different tag (in this case "@domain") and of course implement the proper steps in the domain context of our system.
-To be more descriptive let's imagine that we want to check if a country is added properly in two ways. First we are checking if the adding works via frontend, so we are implementing steps that are clicking, opening pages, filling fields on forms and similar, but also we want to check this action regardlessly of the frontend, for that we need the domain, which allows us to perform actions only on objects.
+Pay attention to the form of these sentences. From the developer point of view they are hiding the details of the feature's implementation.
+Instead of describing "When I click on the select box And I choose France from the dropdown Then I should see the France country in the table"
```

Comment:
Maybe we could format these three sentences to be more consistent with the previous example?